### ISSUE TITLE
Make file extensions consistent

### ISSUE THREAD

**Issue by dleehr** [matthewhirschey/ddh.org#7]
I noticed when building the makefile in #5 that some of the scripts use the `Rdata` extension and some use `RData`. Which is preferred?

### .Rdata

```
generate_depmap_data.R:save(achilles, file = here::here("data", paste0(release, "_achilles.Rdata")))
generate_depmap_data.R:save(expression, file = here::here("data", paste0(release, "_expression.Rdata")))
generate_depmap_data.R:save(expression_id, file = here::here("data", paste0(release, "_expression_id.Rdata")))
generate_depmap_data.R:save(achilles_cor, file = here::here("data", paste0(release, "_achilles_cor.Rdata")))
generate_depmap_pathways.R:load(file = here::here("data", paste0(release, "_achilles_cor.Rdata")))
generate_depmap_stats.R:load(file = here::here("data", paste0(release, "_achilles_cor.Rdata")))
generate_depmap_tables.R:load(file = here::here("data", paste0(release, "_achilles_cor.Rdata")))
```

### .RData
```
app.R:  # Read gene_summary saved as RData using: save(gene_summary, file=here::here("data", "gene_summary.RData"))
app.R:  load(here::here("data", "gene_summary.RData"), envir=tmp.env)
app.R:load(file=here::here("data", "achilles.RData"))
app.R:load(file=here::here("data", "achilles_cor.RData"))
app.R:load(file=here::here("data", "expression_join.RData"))
app.R:load(file=here::here("data", "master_bottom_table.RData"))
app.R:load(file=here::here("data", "master_top_table.RData"))
app.R:load(file=here::here("data", "master_positive.RData"))
app.R:load(file=here::here("data", "master_negative.RData"))
create_gene_summary.R:gene_summary_output_filename <- "gene_summary.RData"
generate_depmap_pathways.R:load(file = here::here("data", "gene_summary.RData"))
generate_depmap_pathways.R:save(master_positive, file=here::here("data", "master_positive.RData")) #change file name to include decX
generate_depmap_pathways.R:save(master_negative, file=here::here("data", "master_negative.RData")) #change file name to include decX
generate_depmap_tables.R:load(file = here::here("data", "gene_summary.RData"))
generate_depmap_tables.R:save(master_top_table, file=here::here("data", "master_top_table.RData"))
generate_depmap_tables.R:save(master_bottom_table, file=here::here("data", "master_bottom_table.RData"))
```

**Comment by matthewhirschey** [comment]
I think that is cutting/pasting errors on my part. *.RData appears to be the 'official' R file format. I will update the Rdata files, and then close this.

**Comment by matthewhirschey** [comment]
.Rdata <- .rdata,
.Rds <- .rds

**Comment by dleehr** [comment]
Thanks matt. FYI I didn't just create this to be pedantic, but it's a good thing to create an issue for - we don't need to do it urgently but we can document what is to be done and get to it when we have a chance. And this one is pretty straightforward and a good way to use branches, pull requests, and github integrations.

For example, if you add "Fixes #7" to your commit message when fixing it, pushing that commit to master will automatically close this issue and link back to the code that fixed it.

**Comment by matthewhirschey** [comment]
Nice. didn't realize that. Thx!
